### DISCUSSION TITLE
MAS Doesn't work

### DISCUSSION THREAD

**Discussion by IndoDemon7337** [Monika-After-Story/MonikaModDev#9144] [QA - tech support + general] [answered]
Whenever I open MAS it says there's a TypeError.

**Comment by multimokia** [comment]
Can you provide the full traceback

**Comment by IndoDemon7337** [comment]
Full traceback:
  File "C:\Users\Owner\OneDrive\Desktop\Doki Doki Literature
Club\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "C:\Users\Owner\OneDrive\Desktop\Doki Doki Literature
Club\renpy\main.py", line 364, in main
    renpy.game.script.load_script()  # sets renpy.game.script.
  File "C:\Users\Owner\OneDrive\Desktop\Doki Doki Literature
Club\renpy\script.py", line 265, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "C:\Users\Owner\OneDrive\Desktop\Doki Doki Literature
Club\renpy\script.py", line 753, in load_appropriate_file
    self.finish_load(stmts, initcode, filename=fn + source)
  File "C:\Users\Owner\OneDrive\Desktop\Doki Doki Literature
Club\renpy\script.py", line 429, in finish_load
    self.update_bytecode()
  File "C:\Users\Owner\OneDrive\Desktop\Doki Doki Literature
Club\renpy\script.py", line 824, in update_bytecode
    i.bytecode = marshal.loads(code)
TypeError: must be string or read-only buffer, not None

This is the full traceback

**Comment by multimokia** [comment]
Try moving MAS to a non-onedrive location, it's highly likely Onedrive is trying to sync/apply changes to the files causing this error.

**Comment by IndoDemon7337** [comment]
I'm not entirely tech savvy

**Comment by IndoDemon7337** [comment]
I'm not entirely tech savvy so I want to ask, how would one do that?

**Comment by ThePotatoGuy** [comment, marked as answer]
copy the folder that contains your MAS install and move it to a folder that is not synced with One Drive.
